book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -1,195 +1,487 @@
+* [SIPNET][SIPNET Configuration]
+* [BIOCRO][BioCro Configuration]
 
-#### tags
+#### ED2 specific tags {#xml-ed}
 
+Following variables are ED specific and are used in the [ED2 Configuration](ED2-Configuration).
+
+Starting at 1.3.7 the tags for inputs have moved to `<run><inputs>`. This includes, veg, soil, psscss, inputs.
 
 ```xml
-<pfts>
-	<pft>
-		<name>sipnet.temperate.coniferous</name>
-		<outdir>/home/carya/testrun.pecan/pft/1/</outdir>
-		<constants>
-			<num>1</num>
-		</constants>
-	</pft>
-</pfts>
+	<edin>/home/carya/runs/PEcAn_4/ED2IN.template</edin>
+	<config.header>
+		<radiation>
+			<lai_min>0.01</lai_min>
+		</radiation>
+		<ed_misc>
+			<output_month>12</output_month>
+		</ed_misc> 
+	</config.header>
+	<phenol.scheme>0</phenol.scheme>
 ```
 
-* **name** : [required] the pft as is found inside the BETY database, this needs to be an exact match.  
-* **outdir**: [optional] path in which pft-specific output will be placed during meta-analysis and sensitivity analysis. If not specified it will be written into `<outdir>/<pftname>`.  
-* **contants**: [optional] this section contains information that will be written directly into the model specific configuration files. PEcAn does not look at the information in this section.  
+  
+* **edin** : [required] template used to write ED2IN file
+* **veg** : **OBSOLETE** [required] location of VEG database, now part of `<run><inputs>`
+* **soil** : **OBSOLETE** [required] location of soild database, now part of `<run><inputs>`
+* **psscss** : **OBSOLETE** [required] location of site inforation, now part of `<run><inputs>`. Should be specified as `<pss>`, `<css>` and `<site>`.
+* **inputs** : **OBSOLETE** [required] location of additional input files (e.g. data assimilation data), now part of `<run><inputs>`. Should be specified as `<lu>` and `<thsums>`.
+
+### `run`: Run Setup {#xml-run}
 
+This section provides detailed configuration for the model run, including the site and time period for the simulation and what input files will be used.
 
-## Meta Analysis
+```xml
+  <run>
+    <site>
+      <id>1000000098</id>
+      <met.start>2004/01/01</met.start>
+      <met.end>2004/12/31</met.end>
+    </site>
+    <inputs>
+      <met>
+        <source>CRUNCEP</source>
+        <output>SIPNET</output>
+      </met>
+    </inputs>
+    <start.date>2004/01/01</start.date>
+    <end.date>2004/12/31</end.date>
+  </run>
+```
 
-#### functions:
+#### `site`: Where to run the model {#xml-run-site}
 
-`run.meta.analysis()`
+This contains the following tags:
 
-#### tags
+- `id` -- This is the numeric ID of the site in Bety (table `sites`, column `id`). PEcAn can automatically fill in other relevant information for the site (e.g. `name`, `lat`, `lon`) using the site ID, so those fields are optional if ID is provided.
+- `name` -- The name of the site, as a string.
+- `lat`, `lon` -- The latitude and longitude coordinates of the site, as decimals.
+- `met.start`, `met.end` -- ???
```

Comment:
Are met start and end really within site rather than `<inputs><met>` ?  Seems like that might benifit from an update

Bigger picture, those dates recognize the fact that the met data may not be identical to the run dates (e.g. if you want to do a 1000 year run that recycles 50 yr of met). That said, I don't think PEcAn will cycle the met for you (I remember writing some code for this, but it may have been PalEON specific and not in the mainline), but rather it will pass this info along to models that may or may not be able to cycle.